acceptance-tests/src/test/kotlin/com/chutneytesting/acceptance/AcceptanceTests.kt

```diff
@@ -0,0 +1,275 @@
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.testcontainers.containers.GenericContainer
+import org.testcontainers.junit.jupiter.Testcontainers
+import org.testcontainers.utility.MountableFile
+import java.io.File
+import java.nio.file.Files
+import java.time.Duration
+
+@Testcontainers
+class AcceptanceTests {
+
+  companion object {
+    private const val ENVIRONMENT_NAME = "DEFAULT"
+
+    private var adminServerInfo: ChutneyServerInfo? = null
+    private var chutneyServer: GenericContainer<Nothing>? = null
+    private var environment: ChutneyEnvironment = ChutneyEnvironment(ENVIRONMENT_NAME)
+
+    private var actionHttpPort: Int? = null
+    private var actionAmqpPort: Int? = null
+    private var actionJakartaPort: Int? = null
+
+    @JvmStatic
+    @BeforeAll
+    fun setUp() {
+      val tempDirectory = Files.createTempDirectory("chutney-acceptance-test-")
+      val memAuthConfigFile = File(AcceptanceTests::class.java.getResource("/blackbox/application-mem-auth.yml")!!.path)
+
+      // Copy mem-auth config
+      Files.copy(memAuthConfigFile.toPath(), tempDirectory.resolve("application-mem-auth.yml"))
+
+      // Start server
+      chutneyServer = GenericContainer<Nothing>("ghcr.io/chutney-testing/chutney/chutney-server:latest").apply {
+        withStartupTimeout(Duration.ofSeconds(80))
+        withExposedPorts(
+            8443, // Chutney
+        )
+        withNetwork(network)
+        withCopyFileToContainer(
+            MountableFile.forClasspathResource("/blackbox/"),
+            "/config"
+        )
+      }
+      actionHttpPort = SocketUtils.freePortFromSystem()
+      actionAmqpPort = 5672
+      actionJakartaPort = SocketUtils.freePortFromSystem()
+      org.testcontainers.Testcontainers.exposeHostPorts(
+        actionHttpPort!!,
+        actionAmqpPort!!,
+        actionJakartaPort!!
+      )
+
+      chutneyServer!!.start()
+      adminServerInfo = ChutneyServerInfo("https://${chutneyServer?.host}:${chutneyServer?.firstMappedPort}", "admin", "admin", null, null, null)
+
+      // Set authorizations
+      val roles = AcceptanceTests::class.java.getResource("/blackbox/roles.json")!!.path
+      HttpClient.post<Any>(adminServerInfo!!, "/api/v1/authorizations", File(roles).readText())
+
+      environment = ChutneyEnvironment(name = ENVIRONMENT_NAME, targets =
+      listOf(
+          ChutneyTarget("CHUTNEY_LOCAL", "https://${chutneyServer?.host}:${chutneyServer?.getMappedPort(8443)}", mapOf("username" to "admin", "password" to "admin")),
+          ChutneyTarget("CHUTNEY_LOCAL_NO_USER", "https://${chutneyServer?.host}:${chutneyServer?.getMappedPort(8443)}", emptyMap()))
+      )
+    }
+
+    @JvmStatic
+    @AfterAll
+    fun cleanUp() {
+      chutneyServer?.stop()
+    }
+  }
+
+  @Test
+  fun `Execution by campaign id with 2 scenarios`() {
+    Launcher().run(executeCampaignById, environment)
```

Comment:
Maybe use the `listOf(...).forEach(...)` idiom here and there to be more readable ?
Are we sure that `Launcher().run(...)` is never throwing Exceptions ?